framework/wazuh/core/exception.py

```diff
@@ -26,8 +26,7 @@ class WazuhException(Exception):
         # Wazuh: 0999 - 1099
         999: 'Incompatible version of Python',
         1000: {'message': 'Wazuh Internal Error',
-               'remediation': 'Please, check `/var/log/wazuh-server/cluster.log` and '
-                              '`/var/log/wazuh-server/logs/api.log` to get more information about the error'},
+       '       remediation': 'Please, check the Wazuh Server logs for more information about the error'},
```

Comment:
I would prefer just calling it `server` instead of `Wazuh Server` since the context already implies it's from Wazuh

```suggestion
       '       remediation': 'Please, check the server logs for more information about the error'},
```

Also, exception 1020 should be removed